diff --git a/geonode/layers/search_indexes.py b/geonode/layers/search_indexes.py
index fb7e363..a615a98 100644
--- a/geonode/layers/search_indexes.py
+++ b/geonode/layers/search_indexes.py
@@ -34,6 +34,9 @@ class LayerIndex(indexes.SearchIndex, indexes.Indexable):
     csw_wkt_geometry = indexes.CharField(model_attr="csw_wkt_geometry")
     detail_url = indexes.CharField(model_attr="get_absolute_url")
     owner__username = indexes.CharField(model_attr="owner", faceted=True, null=True)
+    city = indexes.CharField(model_attr="city", faceted=True, null=True)
+    dataprovider_name = indexes.CharField(model_attr="dataprovider_name", faceted=True, null=True)
+    dataprovidertype = indexes.CharField(model_attr="dataprovidertype", faceted=True, null=True)
     is_published = indexes.BooleanField(model_attr="is_published")
     featured = indexes.BooleanField(model_attr="featured")
     popular_count = indexes.IntegerField(
