diff --git a/haystack/backends/elasticsearch_backend.py b/haystack/backends/elasticsearch_backend.py
index 64d96d7..0558e8b 100644
--- a/haystack/backends/elasticsearch_backend.py
+++ b/haystack/backends/elasticsearch_backend.py
@@ -517,6 +517,7 @@ class ElasticsearchSearchBackend(BaseSearchBackend):
         if end_offset is not None and end_offset > start_offset:
             search_kwargs['size'] = end_offset - start_offset
 
+        print(search_kwargs)
         try:
             raw_results = self.conn.search(body=search_kwargs,
                                            index=self.index_name,
@@ -525,7 +526,7 @@ class ElasticsearchSearchBackend(BaseSearchBackend):
         except elasticsearch.TransportError as e:
             if not self.silently_fail:
                 raise
-
+            
             self.log.error("Failed to query Elasticsearch using '%s': %s", query_string, e, exc_info=True)
             raw_results = {}
 
