diff --git a/geonode/api/resourcebase_api.py b/geonode/api/resourcebase_api.py
index aebda33..cdc1c6a 100644
--- a/geonode/api/resourcebase_api.py
+++ b/geonode/api/resourcebase_api.py
@@ -317,7 +317,7 @@ class CommonModelApi(ModelResource):
         owner = parameters.getlist("owner__username__in")
 
         # Sort order
-        sort = parameters.get("order_by", "relevance")
+        sort = parameters.get("order_by", "-_score")
 
         # Geospatial Elements
         bbox = parameters.get("extent", None)
@@ -458,7 +458,7 @@ class CommonModelApi(ModelResource):
                 "-popular_count")
         else:
             sqs = (
-                SearchQuerySet() if sqs is None else sqs).order_by("-date")
+                SearchQuerySet() if sqs is None else sqs).order_by("-_score")
 
         return sqs
 
